src/components/Pages/Location/LocationServiceList.js

```diff
@@ -24,26 +24,43 @@ const DayHours = ({ day, hours }) => {
   );
 };
 
-const HoursExceptions = ({exceptions}) => {
+const HoursExceptions = ({
+  serviceHoursExceptions,
+  locationHoursExceptions,
+  hoursSameAsLocation,
+}) => {
   const intl = useIntl();
+  let exceptions = serviceHoursExceptions;
+  if (hoursSameAsLocation) {
+    exceptions = locationHoursExceptions;
+  }
   return (
     <div>
       <div className="coa-ContactHoursExceptionsTitle">
-        {intl.formatMessage(i18nContact.exceptions)}
-      </div>
-      <div className="coa-ContactHoursExceptions">
-        {exceptions}
+        {!!exceptions && (intl.formatMessage(i18nContact.exceptions))}
       </div>
+      <div className="coa-ContactHoursExceptions">{exceptions}</div>
```

Comment:
if there are no exceptions, the message will not show up but does the empty div still appear?

Reply:
oh maybe we need the empty div to be there for spacing reasons?

Reply:
good question... !